resume/src/components/Courses.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+import '../App.css';
+
+function Courses(props){
```

Comment:
Since this only represents once course in the ```courses``` array in the JSON resume schema, change this name to just ```Course```

Reply:
and ofc reflect that in the Education component!